content/404.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  Header, MetaPageHead,
+} from './content.types.ts';
+
+    interface NotFoundPageContent {
+      meta: MetaPageHead
+      page: Header;
+    }
```

Comment:
Hey @GabrielEger2! Seems we have some extra spaces here.

Reply:
missing a semi colon on line 6

Reply:
That's odd, I'm pretty sure I had ESLint turned on. I'll fix it ASAP